plugin/audit_log/audit_log.c

```diff
@@ -264,6 +286,22 @@ char *escape_string(const char *in, size_t inlen,
     if (endptr)
       *endptr= out + 1;
   }
+  if (full_outlen)
+  {
+    char tmp[20];
+    in= in + inlen;
```

Comment:
`in == NULL, ful_outlen != NULL` currently can potentially occur for connection events. Not sure why it didn't...